AUTHORS

```diff
@@ -181,7 +181,6 @@ Ryan Petrello <lists@ryanpetrello.com>
 Fuji Goro <gfuji@cpan.org>
 Siddharth Mahendraker <siddharth_mahen@hotmail.com>
 Dave Pacheco <dap@joyent.com>
-Mathias Buus <m@ge.tt>
```

Comment:
How would you guys like me to proceed with this? We can just replace the older entry with the newer one and delete the newer one down the line. This way we maintain the order of contribution and reflect the most recent setup by a user. My thoughts.